docker-compose.yml

```diff
@@ -33,23 +31,17 @@ services:
       - 5432:5432
     volumes:
       - scraper-api-pgdata:/var/lib/postgresql/data
-    networks:
-      - webnet
   redis:
     container_name: scraper-api-redis
     image: redis:6.2-alpine
     restart: always
     ports:
       - 6379:6379
-    command: ["redis-server", "--appendonly", "yes"]   
+    command: ["redis-server", "--appendonly", "yes"]
     environment:
       REDIS_PASSWORD: ${REDIS_PASSWORD}
-    networks:
-      - webnet
-    volumes: 
+    volumes:
       - scraper-api-redis-volume:/data
-networks:
-  webnet:
```

Comment:
Did you remove the network in order to allow the DB to be accessed by the BE app started outside `docker-compose`?

Reply:
I had `ECONNREFUSED` errors in the BE container when started via `docker-compose`. I could fix it by removing this and using the default approach as described here https://docs.docker.com/compose/networking/.